keps/sig-node/20190717-seccomp-ga.md

```diff
@@ -289,9 +365,13 @@ To raise awareness of annotation usage (in case of old automation), an additiona
 will be added when a pod is created with a seccomp annotation:
 
 ```
-warning.kubernetes.io/seccomp: "Seccomp set through annotations. Support will be dropped in v1.22"
+warning.kubernetes.io/seccomp: "Seccomp set through annotations. Support will be dropped in v1.23"
 ```
 
+Note this will be reviewed during implementation to be aligned with the new 
```

Comment:
nit: this link won't work, I think it needs to be `/keps/sig-api-machinery/1693-warnings`

Reply:
Done